### PR TITLE
Remove Sub and SubAssign from Interests

### PR DESCRIPTION
**Pull request by Thomasdezeeuw**
/cc @FXTi

### CODE REVIEW

**Comment by FXTi** [comment]
It looks good, should be right.

File: src/event_imp.rs [hunk 1]
```diff
@@ -804,29 +804,6 @@ impl ops::BitOr for Interests {
     }
 }
 
-impl ops::BitOrAssign for Interests {
```
Comment (carllerche) [inline src/event_imp.rs]: If we keep `BitOr` (which I think we are), we probably can keep `BitOrAssign` as well. WDYT?

Comment (Thomasdezeeuw) [reply src/event_imp.rs]: Sure I'll drop that commit.

Comment (Thomasdezeeuw) [reply src/event_imp.rs]: I've dropped the commit.

**Pull request merged**